build/test.sh

```diff
@@ -51,8 +51,8 @@ fi
 echo
 
 check_dependencies() {
-    # Check if minio is already deployed
-    if helm status minio -n minio > /dev/null 2>&1 ; then
+    # Check if minio is already deployed. We suppress only StdOut for watching error if check will fail.
```

Comment:
```suggestion
    # Check if minio is already deployed. We suppress only `stdout` and not `stderr` to make sure we catch errors if `helm status` fails
```

If you think its better.